ehitchhiking-ui/src/styles/constants.sass

```diff
@@ -3,3 +3,4 @@ $default-padding: 5%
 $trip-card-height: 4.5rem
 $height-empty-block: 10px
 $number-of-trips-in-list: 4
+$height-borders: 2px
```

Comment:
bad naming: height-borders

Reply:
changed to $height-borders-of-trip-in-list